boards/arm/efr32_slwstk6061a/efr32_slwstk6061a_defconfig

```diff
@@ -7,6 +7,7 @@ CONFIG_BOARD_EFR32_SLWSTK6061A=y
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
+CONFIG_SPI=y
```

Comment:
Despite "Default Board Configuration" guidelines requirement to enable all major components on the evaluation board there are sample applications, like `samples/basic/minimal/` which expect default configuration not to enable drivers which are not required. Maybe we should remove this option from `_defconfig` files. Sample application that require SPI enable `CONFIG_SPI` in their respective `prj.conf` file.

Reply:
done, but i noted that a few other boards (about 10) also enable spi in their board config.

Reply:
Indeed, the approach to configuring peripherals is still a bit inconsistent.